.github/workflows/scripts/geth-commit.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# This script finds the base commit corresponding to a geth tag
+# from which libevm was last based on, and returns it to the caller.
+# If an error occurs, this one is echo-ed and the script exits with code 1.
+
+git remote add geth https://github.com/ethereum/go-ethereum.git
+git fetch geth 'refs/tags/*:refs/tags/*'
+
+geth_tags=$(git ls-remote --tags --sort=-version:refname geth "v[1-9]*" | awk '{print $2}' | sed 's/refs\/tags\///')
+
+base_geth_tag_hash=
+for geth_tag in $geth_tags; do
+  geth_tag_hash="$(git rev-parse --short $geth_tag)"
```

Comment:
(nit) There's no benefit to `--short` here and a (granted near-negligible for these purposes) con.

```suggestion
  geth_tag_hash="$(git rev-parse $geth_tag)"
```

Reply:
Actually.... I set it to short to be able to quickly look it up (CTRL/CMD+F) in the git commit log on Github which only shows short commit hashes in the list 😄 But yes, it sounds strange to have it, so removing it!